webpack.config.client.js

```diff
@@ -117,7 +117,7 @@ module.exports = ({ resolvePath, IS_CI, IS_PROD, START_DEV_SERVER }) => {
        * we need it, without importing it in every file or importing
        * every locale.
        */
-      new webpack.ContextReplacementPlugin(/moment[/\\]locale$/, /en-gb/),
+      new webpack.ContextReplacementPlugin(/moment[/\\]locale$/, /en-gb|fa|yo/),
```

Comment:
Does this mean we will have to at some point include all the service locales in the list? Could we use `webpack.IgnorePlugin`

Reply:
Yeah, this works well IMO https://github.com/bbc/simorgh/blob/BundleSplitting6/webpack.config.client.js#L116-L119